src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -161,7 +161,7 @@
 Tool '{1}' (version '{2}') was successfully installed.</value>
   </data>
   <data name="LocalToolInstallationSucceeded" xml:space="preserve">
-    <value>You can invoke the tool from this directory using the following command: dotnet tool run {0}
+    <value>You can invoke the tool from this directory using the following command: dotnet tool run {0} or dotnet {0}
```

Comment:
@KathleenDollard I added "or dotnet {0}" to match you can both invoke it from `dotnet tool run` and `dotnet `